Bug: Executor curl - nil pointer

**Describe the bug**

I've followed instructions from docs https://docs.testkube.io/test-types/executor-curl/

Log from executor pod: 

```
{"type":"line","content":"HTTP/2 403 \r\n","time":"2023-04-04T09:15:18.825356559Z"}
{"type":"line","content":"date: Tue, 04 Apr 2023 09:15:18 GMT\r\ncontent-type: text/html; charset=UTF-8\r\ncross-origin-embedder-policy: require-corp\r\ncross-origin-opener-policy: same-origin\r\ncross-origin-resource-policy: same-origin\r\npermissions-policy: accelerometer=(),autoplay=(),camera=(),clipboard-read=(),clipboard-write=(),fullscreen=(),geolocation=(),gyroscope=(),hid=(),interest-cohort=(),magnetometer=(),microphone=(),payment=(),publickey-credentials-get=(),screen-wake-lock=(),serial=(),sync-xhr=(),usb=()\r\ncache-control: private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0\r\nexpires: Thu, 01 Jan 1970 00:00:01 GMT\r\nreport-to: {\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=j0GFDfe6o4Rso5WOaWWkObMBaa1SKcz7EV57yFAtWHL1xqCa4QyQ4DK0w8gB1yfafP%2B1srqWAfb2t18coShY80RN6NJ4fV%2FxqfLKU%2BRytwdM7Djysn2b2B17ZSM%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}\r\nnel: {\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}\r\nexpect-ct: max-age=86400, enforce\r\nreferrer-policy: same-origin\r\nx-content-type-options: nosniff\r\n","time":"2023-04-04T09:15:18.825655783Z"}
{"type":"line","content":"x-frame-options: SAMEORIGIN\r\nx-xss-protection: 1; mode=block\r\nserver: cloudflare\r\ncf-ray: 7b2872329b49b15d-ATL\r\n\r\n\u003c!DOCTYPE html\u003e\n\u003chtml lang=\"en-US\"\u003e\n\u003chead\u003e\n    \u003ctitle\u003eJust a moment...\u003c/title\u003e\n    \u003cmeta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\"\u003e\n    \u003cmeta http-equiv=\"X-UA-Compatible\" content=\"IE=Edge\"\u003e\n    \u003cmeta name=\"robots\" content=\"noindex,nofollow\"\u003e\n    \u003cmeta name=\"viewport\" content=\"width=device-width,initial-scale=1\"\u003e\n    \u003clink href=\"/cdn-cgi/styles/challenges.css\" rel=\"stylesheet\"\u003e\n    \n\n\u003c/head\u003e\n\u003cbody class=\"no-js\"\u003e\n    \u003cdiv class=\"main-wrapper\" role=\"main\"\u003e\n    \u003cdiv class=\"main-content\"\u003e\n        \u003cnoscript\u003e\n            \u003cdiv id=\"challenge-error-title\"\u003e\n                \u003cdiv class=\"h2\"\u003e\n                    \u003cspan class=\"icon-wrapper\"\u003e\n                        \u003cdiv class=\"heading-icon warning-icon\"\u003e\u003c/div\u003e\n                    \u003c/span\u003e\n                    \u003cspan id=\"challenge-error-text\"\u003e\n                        Enable JavaScript and cookies to continue\n                    \u003c/span\u003e\n                \u003c/div\u003e\n            \u003c/div\u003e\n        \u003c/noscript\u003e\n        \u003cdiv id=\"trk_jschal_js\" style=\"display:none;background-image:url('/cdn-cgi/images/trace/managed/nojs/transparent.gif?ray=7b2872329b49b15d')\"\u003e\u003c/div\u003e\n        \u003cform id=\"challenge-form\" action=\"/echo/get/json?__cf_chl_f_tk=TgpYounAMVJ5kn2Zcbb_pJtfpnTvEYKnKofB2mIe9So-1680599718-0-gaNycGzNCpA\" method=\"POST\" enctype=\"application/x-www-form-urlencoded\"\u003e\n            \u003cinput type=\"hidden\" name=\"md\" value=\"3Hp1pQarzVO6FO1JWRrmL90NbNu4dv0.zbgXPcPa8ug-1680599718-0-AeqrL1RSSEwiTsEvPG8pGrjr1U_kHx43mkNfnGdC6MRHkwRiyIS5QInj4OMVQKhENpkdvV7DtKWHWqtwEuAW6Zb86_V4XGBsirQWXB_SLIAwuVv5lKUzH1pWN2M4iW6dBMrafScpjcYrf79OfC03oiTCgP46S20aRSWnf-qm3O0N3uhq8_C2yg1v15tFEP5eMry61cr7JWw6R7cTbzI9eBCdn_6tmd05kpJ3W4rvJL9VVw6XvAUiKa47_SMOvnsvO7gmJ0xNcKu9HNaxNkJRzKcEoYamRTQnbLKndHeNq_cgG4Q_MABz5fTo6ZhL2i5FSCYwjVl8ZlmRa3Snsx2_duS-uvSPgA3gYYcwnY19Xn0r--GhY0ilB4D_u4JLgr4PyywUhBz0k2MyG7qS5iNXWejicuMnsmp5QRb0aGh-B8x-1","time":"2023-04-04T09:15:18.825887809Z"}
{"type":"line","content":"jSsMlMxmyfnt8rBcC5BSJHn4uFlyuAZfYbV55LBJRMn2aGJO69wuJeEI21XjD71BhHv8ALlgKmYLzuAST0_vCHTlS_jcKI24JB3_Z0CSusvPG-qhxawpC9BzaOMOmND0nnwLwNeBQ_7owhaZ5knc7GoCoT_b1QhczUhypnVFIztxf0M4VjxguBEj7_3nM8pgBWdFzU8DZnso6w3LZhXKW6KRK2LYt3eprlsscvEdm-Nd4h-s_YBqrozDlSJOGrQ0IiCRGHAXFTjvFgMG1049LKGCwatoYxKl9N5VA7yKJZDhzaIaqrmDWb04-erXaTkZSRc6izNmHc-FfCycRLAi5Fg7_yoeq4x6789_HUTkwwLl5VdacwqrSj3pqFVyMtyyL-gX5-iE686XE-ayQu7E1ZiR39mD5vn_AJu7Pf7Ki1KpMH7798zXpZtzvmtCXT308f803aqo735YMf5Gl5ruIBgdZKzYsDbWPKhuUEpczMbYEyc5exbRSoMbZsYqCl3jQ_CdinzTULReWe40bEBDYL6JhLeWSahbICz-WgtBJm4bNAMCaHwrwAMaLaI25zgEPEoeIUmYJ2J1PgSYwTLe7XFeXDSpYaBNNgnGrCre_19zgxPC1CWal0PYBjgrr36J_7C11fGkiLaxSgL6jAHUkWJ5HX9s4IP0RMAjehVIcnrnwl1vsMNwkjFzogxaUM5l8Jft6VnF3qjKF5NcCq3z-lgkcJsQuzy7-WwzxsgQt2cYowp3jnyhw7zUZENlggBwSsJzozILa5gmjpB24RGk7K6-A9JFzDnj9eOrvQ5bi7boW3QdoRmjdi7_g09kt6ykmrjzzDReIRmYDwNwHwZHvPfA4ZZBldrfSisix7djpTubGJ34tLykexuw-MsRwgXTVJt3iX1ab6u-mStebmsg5dZykbuvBYlPcFG3KzadU1JG3F1UZiwbcP51mpQVMbYrv86hYesjE4gsSgOAMEkJrZpBT98pWCMO6yveZsgsV3DR9TC8z9RwQObDh6dlmlZbtZR6tTQJKY1DWWZ1oEIRvQuvOLEFMSOJ3dup60qPEkK3TrOr3IDOrm9mhhnz2lfzflFW-oDFDzPtfv7EUm91NRosXwC-C86HgzX4aCi55uzW5eL9doxOJgr5gr9WKXZ_r5-Zxo3g899PPglY-GVYeRtf07C-4FBAZgDtsGiQsuxDVI3Q9Esk4yX6aLGpPCbBn1J7ImkjkGBK_5JKTeobb82SbMHPR9Gcum0SNWzFlNs1NH6eiAD4QDN4EGcdg2TBtJx8kVzhFomXR4H-LVqNMsC0tLC-eaZOqBCendsTlPh0G_-ov1zOW9Un57UUPHXC7_9Hyu2O785ueemBZ8cLctfjM","time":"2023-04-04T09:15:18.828207481Z"}
{"type":"line","content":"HXZ7u0xRYmXlmfWpjqA9Hmv4qTiI6Dhj2Ish8C4RONn0gs_DOl8yp-aEYFI5O_I1u6xOK1UAKf4scMyz4KUfub7qZhHmRCkzhBwteHu9b5z5zUcLtXS-n99VDCadLKX5Xj9oBXwJXZ1vwiOS6ppBF7wYMuoHKS0K6lTD9tnT2mC2GVvJlEY0ioXeCwz35NJarmlQ6vAy9JtpcwVVgbhx5QSYCbux9KadWdI8bIMggLQwntOyhDpRiOIKoA0TrINWW02yBhEkClVqHMybdOX95RqrnLI83WYb0fk6Pis-Ze7sbIQdu6Fh0NDz_e9uq6muNJMvBPdUGUDhbthj3KIjRzwu8ppY1CvqaAs3w8ee-r9VjnMT1-jRkk7r29m6KO2lJ7lXmfo4AQ25rSjSCB49YUCZoiegr10WlNE7moDVVg3uB-ZsSDbHiPtbP-3EuTrx2ccaFBwFaCG_coGmymGgefhv-LfdGUcRgbJst93-g9Wt7YiwMh5xdm2wLCFKSZFLjyXiG0fHp2eVnr14GvDEUxQXg4a3X_l5T2KshO2fg6HT0apQKsDqTUiVfPaRAFjx3fnUsgv6BS0YE7OLg0mduybUqG3TrMYMac1bkCGu5gy0DDbtgDJ799uA4gUuMeX4VTDv_eKnh0V9QSoMLFmMJ37qtexI849XSFDaioBBdN7MGBYgsOtDCbNCd_sncUUvcUKsKmIKL5PIohGXnPM4HBWdXUNfjFQjQrVbXPYkySyDjfs9pjI-13LlN3zOfxiLDrksQ91bAyTlRHzz8GGgQUuESdwyCu29_s5UqoR33ZTM13ndVr4RTzUSjBx_OZoVL5RvZWfVxhdSNgSi4\"\u003e\n        \u003c/form\u003e\n    \u003c/div\u003e\n\u003c/div\u003e\n\u003cscript\u003e\n    (function(){\n        window._cf_chl_opt={\n            cvId: '2',\n            cZone: 'reqbin.com',\n            cType: 'managed',\n            cNounce: '98058',\n            cRay: '7b2872329b49b15d',\n            cHash: '9d43d712b4523c5',\n            cUPMDTk: \"\\/echo\\/get\\/json?__cf_chl_tk=TgpYounAMVJ5kn2Zcbb_pJtfpnTvEYKnKofB2mIe9So-1680599718-0-gaNycGzNCpA\",\n            cFPWv: 'b',\n            cTTimeMs: '1000',\n            cMTimeMs: '0',\n            cTplV: 5,\n          ","time":"2023-04-04T09:15:18.830763357Z"}
{"type":"line","content":"  cTplB: 'cf',\n            cK: \"\",\n            cRq: {\n                ru: 'aHR0cHM6Ly9yZXFiaW4uY29tL2VjaG8vZ2V0L2pzb24=',\n                ra: 'Y3VybC83Ljg4LjE=',\n                rm: 'R0VU',\n                d: 'qUx66q6P32mHEAJfdi3yUNfRVax/1btKVqMv9bcztpzkkbaC9df0Xhh+I6GYI0LuxJ5TqWocDJ5BhW6gw9ZpaXhvfFP9VXF6QKnR2swXfTKHug9jqCfHoDrNgZiG/wVKAChSZ0OwCagnVn/5+WYq5g0V/HE3m7Ybi43ASZxBeVupCFyyMfTGkYErVAU3fI9X/9gmH/XL0aPb0vuP6GhtvjYpYe9URwY5HO1zRF0/XSyhjDMSd/YNhItmaFQOR0CsF9a12Ra9M04Tpeb9aQtzb+etzp22L/yyoRi7eGiSPle9dGMmv99cCWq1X1R2smuyzxCO4Yuo2lWZpIpFq3eAapQJ+/7cVE7u9jFdMMJBhYay8J8SdMDIOvKx4qnG59JVvAw3lKdZzhbymNydLMa9jIljrkk7qpZQMjhxnhIRdeQDcUVmI4t1p/Ru0K5t1q1/MKgMsBgO5C2rhV8HwIrxCUfaHr80mxDQu127i1Vg8O6ii7ntpWXcFv92O74eV5+lwGMTu2GQ3blanoZPA6Zen8M5vHjfmcVU3yGTPp5M+E8PZAac/wvtWrblQmhMFjmwb92rRB7Qcvy6NaYFAGFD3+AmSez+VLiAeh1GOLFd5cpJsW3ZDg+xB7O/3ndEIuwD',\n                t: 'MTY4MDU5OTcxOC44MTIwMDA=',\n                m: 'uQePBJCZvnCmZ6WXiZOImXR1bIwi+s85HGQpxHDowBE=',\n                i1: 'DgiVGwV2pCKilbrzIH7KnA==',\n                i2: 'AOH0f15MXmAYzJtra4HTwQ==',\n                zh: 'LCW3AZH/FGYMtoptFGg2LyVk23ScKwRlpKwnq4Z30Ks=',\n                uh: 'lX+4ytSWIAYmrLLYT6PFt6+dDmxcA8mUI2GojLU4YNU=',\n                hh: 'vrpRu8Ley44OIG8+ikOYvBHgHkkGD06acbUIcsoF9q0=',\n            }\n        };\n        var trkjs = document.createElement('img');\n        trkjs.setAttribute","time":"2023-04-04T09:15:18.832958541Z"}
{"type":"line","content":"('src', '/cdn-cgi/images/trace/managed/js/transparent.gif?ray=7b2872329b49b15d');\n        trkjs.setAttribute('alt', '');\n        trkjs.setAttribute('style', 'display: none');\n        document.body.appendChild(trkjs);\n        var cpo = document.createElement('script');\n        cpo.src = '/cdn-cgi/challenge-platform/h/b/orchestrate/managed/v1?ray=7b2872329b49b15d';\n        window._cf_chl_opt.cOgUHash = location.hash === '' \u0026\u0026 location.href.indexOf('#') !== -1 ? '#' : location.hash;\n        window._cf_chl_opt.cOgUQuery = location.search === '' \u0026\u0026 location.href.slice(0, location.href.length - window._cf_chl_opt.cOgUHash.length).indexOf('?') !== -1 ? '?' : location.search;\n        if (window.history \u0026\u0026 window.history.replaceState) {\n            var ogU = location.pathname + window._cf_chl_opt.cOgUQuery + window._cf_chl_opt.cOgUHash;\n            history.replaceState(null, null, \"\\/echo\\/get\\/json?__cf_chl_rt_tk=TgpYounAMVJ5kn2Zcbb_pJtfpnTvEYKnKofB2mIe9So-1680599718-0-gaNycGzNCpA\" + window._cf_chl_opt.cOgUHash);\n            cpo.onload = function() {\n                history.replaceState(null, null, ogU);\n            };\n        }\n        document.getElementsByTagName('head')[0].appendChild(cpo);\n    }());\n\u003c/script\u003e\n\n\n\u003c/body\u003e\n\u003c/html\u003e\n","time":"2023-04-04T09:15:18.835225646Z"}
{"type":"line","content":"✅ Execution succeeded","time":"2023-04-04T09:15:18.844610451Z"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x1177d46]

goroutine 1 [running]:
github.com/kubeshop/testkube/contrib/executor/curl/pkg/runner.(*CurlRunner).Run(_, {{0xc0000455d8, 0x18}, {0xc00044dd00, 0xf}, {0x0, 0x0}, {0xc00044dcf8, 0x8}, {0xc00044dd10, ...}, ...})
	/go/contrib/executor/curl/build/contrib/executor/curl/pkg/runner/runner.go:134 +0xcc6
github.com/kubeshop/testkube/pkg/executor/agent.Run({0x15a1fb0, 0xc0000f6b60}, {0xc00003a040, 0x2, 0x60?})
	/go/contrib/executor/curl/build/pkg/executor/agent/agent.go:67 +0x53e
main.main()
	/go/contrib/executor/curl/build/contrib/executor/curl/cmd/agent/main.go:18 +0xb4
```

--- a/contrib/executor/curl/pkg/runner/runner.go
+++ b/contrib/executor/curl/pkg/runner/runner.go
@@ -131,7 +131,7 @@ func (r *CurlRunner) Run(execution testkube.Execution) (result testkube.Executio
 	}
 
 	if responseStatus != expectedStatus {
-		outputPkg.PrintLog(fmt.Sprintf("%s Test run failed: cannot process expected status: %s", ui.IconCross, err.Error()))
+		outputPkg.PrintLog(fmt.Sprintf("%s Test run failed: response status don't match expected %d got %d", ui.IconCross, expectedStatus, responseStatus))
 		return *result.Err(fmt.Errorf("response status don't match expected %d got %d", expectedStatus, responseStatus)), nil
 	}
 

